MIP0/mip0.md

```diff
@@ -7,287 +7,252 @@ Title: The Maker Improvement Proposal Framework
 
-**Withdrawn:** when a MIP Author withdraws their MIP proposal, such as when:
+- **Obsolete:** Assigned when:
 
- - A MIP may be withdrawn at any point before it enters the Governance cycle. 
- - Note that a withdrawn proposal can be taken over from the original Author with a simple transition facilitated by a MIP Editor(s) and the respective parties. If the original MIP Author ceases to be available, the MIP Editor(s) may proceed with the transfer of Authors.
+  - A MIP has been superseded or deprecated.
+  - A MIP has been deferred for over six months.
+  - A MIP Author has abandoned the proposal and no person has communicated willingness to take over the responsibility of a MIP Author.
 
-**Deferred:** when a proposal has been deemed as not ready or not a priority but can be re-proposed at a later date.
--   Request for Comments (RFC) - Forum poll/signal request rejects a MIP Proposal.
+#### MIP Status Change Process
 
-**Obsolete:** when a proposal is no longer used or is out of date, such as:
-    
--   A MIP is replaced with a new proposal.
--   A MIP has been deferred for over 6 months.
--   A MIP Author has abandoned the proposal and no person has communicated willingness to take over MIP Author responsibility.
--   A MIP has been replaced by a newer, more updated MIP Proposal.
--   A MIP no longer makes sense to keep in consideration.
-    
-  
-**MIP Status Change Process:**
-    
+If a MIP Author requests a status change for a MIP, a MIP Editor will review it. If the status change is warranted, the MIP Editor will change the status. Otherwise, the MIP Editor will revert and highlight issues for the MIP Author to fix before requesting another status change.
 
-A status change for a MIP is requested by the MIP Author and will be reviewed by the MIP Editor(s) to see if it meets the status criteria of the requested status change. If it does, the Editor(s) will change the status of the MIP and the Author may proceed with the next stage of the process. If it does not, the MIP Editor(s) will revert with highlighted issues, and the Author must fix the highlighted issues before requesting another status change.
-    
 ---
 ### MIP0c4: MIP Components and MIP Component Types
 
 
-**MIP Components**
+#### MIP Components
 
-- When necessary, MIPs can have multiple components if it needs to contain multiple units of logic to satisfy completeness. A MIP can also have only a single component.
-- MIP components are categorized by types, depending on what kind of logic they contain. MIP components are named by their parent MIP. The abbreviation convention MIPXcY is used to refer to these components (as seen in this document).
-- A MIP component has one type or no types. 
+In order to satisfy the Core Principles of clarity and completeness, a MIP may need to have multiple components that each define distinct units of logic.
 
+##### Component Types
```

Comment:
Remove this header imo. No need for headers when you're only making a single related statement.